doc/source/data.rst

```diff
@@ -1,20 +1,104 @@
 Data
 ====
+VerMoUTH knows several data structures, most of which describe MD particles
+(such as atoms or CG beads) and connections between those. As such, these are
```

Comment:
I believe it would be more useful to keep this practical and simple. Something along the lines that we represent a molecule as graphs. Therefore nodes are atoms or beads etc. The use of MD particles seem unnecessarily convoluted.